File: Modul6/Bahasa_Python/PRAK602-2310817210012-Nazmi-Hakim.py
def jumlahzetsu(ruangan, zetsu):
    hasil=[]
    for i in range(ruangan):
        hasil.append(zetsu[i]*(i+1))
    return hasil

ruangan=int(input())
zetsu=list(map(int, input().split()))
hasil= jumlahzetsu(ruangan, zetsu)
print(" ".join(map(str, hasil)))

File: Modul6/Bahasa_C/PRAK602-2310817210012-Nazmi-Hakim.c
#include <stdio.h>

void jumlahzetsu(int ruangan, int zetsu[]) {
    int hasil[ruangan];

    for (int i = 0; i < ruangan; i++) {
        hasil[i] = zetsu[i] * (i + 1);
    }

    printf("%d", hasil[0]);
    for (int i = 1; i < ruangan; i++) {
        printf(" %d", hasil[i]);
    }
    printf("\n");
}

int main() {
    int ruangan;
    scanf("%d", &ruangan);

    int zetsu[ruangan];
    for (int i = 0; i < ruangan; i++) {
        scanf("%d", &zetsu[i]);
    }

    jumlahzetsu(ruangan, zetsu);

    return 0;
}